src/backend/distributed/commands/multi_copy.c

```diff
@@ -809,14 +810,15 @@ OpenCopyConnections(CopyStmt *copyStatement, ShardConnections *shardConnections,
 
 
 /*
- * CanUseBinaryCopyFormat iterates over columns of the relation given in rowOutputState
- * and looks for a column whose type is array of user-defined type or composite type.
- * If it finds such column, that means we cannot use binary format for COPY, because
- * binary format sends Oid of the types, which are generally not same in master and
- * worker nodes for user-defined types.
+ * CanUseBinaryCopyFormat iterates over columns of the relation and looks for a
+ * column whose type is array of user-defined type or composite type. If it finds
+ * such column, that means we cannot use binary format for COPY, because binary
+ * format sends Oid of the types, which are generally not same in master and
+ * worker nodes for user-defined types. Function also error outs if it can not
```

Comment:
This function does not error out. It just returns false if a column does not have binary output function.